src/app/app-routing.module.ts

```diff
@@ -35,12 +35,12 @@ const routes: Routes = [
         component : CardsTopicsexhibitionsComponent
       },
       {
-        path : ':Hardware ',
+        path : ':/Hardware ',
```

Comment:
why was this / added..? and remove the space after Hardware, it will consider space also in url